src/util/curl_easy.cpp

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/curl_easy.hpp"
+
+size_t
+StringWriterCallback(void *contents, size_t size, size_t nmemb, void *buffer) {
+  ((std::string*)buffer)->append((const char*)contents, size * nmemb);
+  return size * nmemb;
+}
+
+size_t
+StreamWriterCallback(void *contents, size_t size, size_t nmemb, void *stream) {
```

Comment:
we use camelCase for functions. At first sight I thought this was a constructor

Reply:
Fixed.